main.go

```diff
@@ -1,32 +1,75 @@
 package main
 
 import (
+	"database/sql"
 	"encoding/json"
+	"fmt"
 	"log"
 	"net/http"
-	"time"
+	"os"
 
+	_ "github.com/go-sql-driver/mysql"
 	"github.com/gorilla/mux"
 )
 
-// TODO: Define Task Struct
-type task struct {
+// Database object for handlers to use
+var db *sql.DB
+
+type Task struct {
 	Description      string `json:"description"`
 	CreatedTimestamp string `json:"createdTimestamp"`
 }
 
-// TODO: Define tasks variable globally
-var tasks = []task{
-	{Description: "Get 10k followers", CreatedTimestamp: time.Now().Format(time.RFC3339)},
-	{Description: "Get paid from content", CreatedTimestamp: time.Now().Format(time.RFC3339)},
+// TODO: Implement API Error Handling
+// type ErrorResponse struct {
+// 	Error   string `json:"error"`
+// 	Message string `json:"message"`
+// 	Detail  string `json:"detail"`
+// }
+
+func getHealth(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusOK)
 }
 
-// TODO: Define getTasks Handler
 func getTasks(w http.ResponseWriter, r *http.Request) {
+	log.Println("attempting query to Task table")
```

Comment:
Capitalize log message to remain log consistency.
```suggestion
	log.Println("Attempting query to Task table")
```